integr8ly/integreatly-operator
# Intly 5437 - User Permission Functional Test

## JIRA
https://issues.redhat.com/browse/INTLY-5437

## About
This change only tests the first two steps of functional test B02 for both `rhmi developer` user and a `dedicated admin` user

## Verification
Note you must be signed into and OSD cluster with a full RHMI install
- Run `make test/functional`
- Ensure tests are successful
